src/renderer/screens/Preferences/keyboard/leds/Brightness.js

```diff
@@ -16,43 +16,65 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-import FormControl from "@mui/material/FormControl";
-import FormHelperText from "@mui/material/FormHelperText";
+import Box from "@mui/material/Box";
+import Skeleton from "@mui/material/Skeleton";
 import Slider from "@mui/material/Slider";
 import Typography from "@mui/material/Typography";
 
-import React from "react";
+import usePluginEffect from "@renderer/hooks/usePluginEffect";
+import React, { useState } from "react";
 import { useTranslation } from "react-i18next";
 
 const Brightness = (props) => {
   const { t } = useTranslation();
-  const { value, onChange, visible } = props;
+  const { registerModifications } = props;
+
+  const [ledBrightness, setLedBrightness] = useState(255);
+
+  const initialize = async (focus) => {
+    let brightness = await focus.command("led.brightness");
+    brightness = parseInt(brightness);
+
+    setLedBrightness(brightness);
+  };
+  const loaded = usePluginEffect(initialize);
 
   const formatValue = (value) => {
     return ((value / 255) * 100).toFixed(0) + "%";
   };
 
-  if (!visible) return null;
+  const onChange = async (event) => {
+    const brightness = event.target.value;
+    await setLedBrightness(brightness);
+    await registerModifications("led.brightness", brightness);
+  };
 
   return (
-    <FormControl sx={{ display: "flex" }}>
-      <Typography gutterBottom>
-        {t("preferences.keyboard.led.brightness.label")}
-      </Typography>
-      <Slider
-        max={255}
-        step={16}
-        marks
-        valueLabelDisplay="auto"
-        valueLabelFormat={formatValue}
-        value={value}
-        onChange={onChange}
-        sx={{ width: "20em" }}
-      />
-      <FormHelperText>
-        {t("preferences.keyboard.led.brightness.help")}
-      </FormHelperText>
-    </FormControl>
+    <Box sx={{ display: "flex" }}>
+      <Box>
```

Comment:
This feels like a fantastic candidate to extract into a jsx component with heading and subheading props.